src/classes/BDI_DataImportDeleteBTN_CTRL.cls

```diff
@@ -42,6 +42,15 @@ public with sharing class BDI_DataImportDeleteBTN_CTRL {
     /** @description Action type specified to delete all Data Import records */
     public static final String ACTION_DELETE_ALL = 'deleteAll';
 
+    /** @description Data Import SObject API name */
+    static final String DATA_IMPORT_API_NAME = 'DataImport__c';
```

Comment:
This can be removed by using the object token to get the object describe result.

Reply:
fixed!